src/lib/homestore_backend/heap_chunk_selector.cpp

```diff
@@ -96,6 +96,50 @@ csharedChunk HeapChunkSelector::select_chunk(homestore::blk_count_t count, const
     return vchunk.get_internal_chunk();
 }
 
+csharedChunk HeapChunkSelector::select_specific_chunk(const chunk_num_t chunkID) {
+    if (m_chunks.find(chunkID) == m_chunks.end()) {
+        // sanity check
+        LOGWARNMOD(homeobject, "No chunk found for ChunkID {}", chunkID);
```

Comment:
`LOGW(...` macro is same as `LOGWARNMOD(homeobject, ...`

Reply:
I noticed this before, but since heap chunk selector is an independent module in HO and macro LOGW(....) is defined in homeobject_impl.hpp and I try to avoid include homeobject_impl.hpp to heap chunk selector to keep it simple and independence.  I think that is why LOGWARNMOD(homeobject,....) is still being widely used in heap chunk selector.